activestorage/app/models/active_storage/preview.rb

```diff
@@ -56,15 +56,18 @@ def image
   # preview has not been processed yet.
   #
   # This method synchronously processes a variant of the preview image, so do not call it in views. Instead, generate
-  # a stable URL that redirects to the short-lived URL returned by this method.
-  def service_url(**options)
+  # a stable URL that redirects to the URL returned by this method.
+  def url(**options)
     if processed?
-      variant.service_url(**options)
+      variant.url(**options)
     else
       raise UnprocessedError
     end
   end
 
+  alias_method :service_url, :url
+  deprecate service_url: :url
```

Comment:
Was `service_url` added to Rails 6.0? If no, we can change this without deprecation.

Reply:
I think `service_url` was available in Rails 5.2 as well: https://apidock.com/rails/v5.2.3/ActiveStorage/Variant/service_url